crates/release_util/src/utils.rs

```diff
@@ -0,0 +1,159 @@
+//! Common utilities for release management in a Git repository.
+//!
+//! This module contains code that is common between this crate and the integration tests crate.
```

Comment:
```suggestion
//! This module contains code that is common between this crate and the integration tests' crate.
```
Or
```suggestion
//! This module contains code that is common between this crate and the integration test's crate.
```

Reply:
I think I actually didn't want a plural or possession here, simplified to "the integration test crate"

Reply:
https://github.com/holochain/release-integration/pull/3/commits/a7090a20f07bf2cc1be9d650f1cbb9a5e7146c0c